homeassistant/components/openweathermap/config_flow.py

```diff
@@ -92,6 +83,26 @@ async def async_step_import(self, import_input=None):
             config[CONF_LANGUAGE] = DEFAULT_LANGUAGE
         return await self.async_step_user(config)
 
+    def _get_schema(self):
+        return vol.Schema(
+            {
+                vol.Required(CONF_API_KEY): str,
+                vol.Required(CONF_NAME, default=DEFAULT_NAME): str,
+                vol.Required(
+                    CONF_LATITUDE, default=self.hass.config.latitude
+                ): cv.latitude,
+                vol.Required(
+                    CONF_LONGITUDE, default=self.hass.config.longitude
+                ): cv.longitude,
+                vol.Required(CONF_MODE, default=DEFAULT_FORECAST_MODE): vol.In(
+                    FORECAST_MODES
+                ),
+                vol.Required(CONF_LANGUAGE, default=DEFAULT_LANGUAGE): vol.In(
```

Comment:
This was optional, thus the `default=...`.

Reply:
yep, now it's required, because integration can't be created without coordinates.

Reply:
The coordinates are already be present in `hass` (I guess in 98 % of all use-cases). Entering them was the fall-back or if you want to monitor a different location. I don't see why this behavior should be changed with the config flow.

Reply:
What if user wants to create new OWM integration for other location.
Why he or she has to create the integration then change location in options?